bin/concert

```diff
@@ -65,8 +65,8 @@ def get_ipython_shell(config=None):
             allow_CTRL_C()
             inputhook_manager._current_gui = 'gevent'
         else:
-            print("IPython {} is too old for gevent integration. "
-                  "Please upgrade IPython or disable gevent.".format(version))
+            print(("IPython {} is too old for gevent integration. "
```

Comment:
This 2to3 is so sophisticated...

Reply:
Yep, I am gonna have some fun with `sed` in some cases.